frontends/web/src/locales/en/app.json

```diff
@@ -1450,6 +1455,7 @@
     "addAccount": "Add account",
     "buy": "Buy crypto",
     "device": "Manage device",
+    "insurance": "Insurance",
```

Comment:
same as above in bisruance.title => if it makes sense add it only once so it does not need to be translated into all languages multiple times.

Reply:
left it in the sidebar (for coherence with other pages) and removed the bitsurance.title one.